src/main/webapp/app/shared/util/firebase/firebase-path-utils.ts

```diff
@@ -1,7 +1,18 @@
 import { Gene } from 'app/shared/model/firebase/firebase.model';
-import { ExtractPathExpressions } from './firebase-crud-store';
 import { FB_COLLECTION } from 'app/config/constants/firebase';
 
+// https://stackoverflow.com/questions/66621546/how-to-extract-path-expression-from-an-interface-in-typescript
+export type ExtractPathExpressions<T, Sep extends string = '/'> = Exclude<
+  keyof {
+    [P in Exclude<keyof T, symbol> as T[P] extends any[] | readonly any[]
+      ? P | `${P}/${number}` | `${P}/${number}${Sep}${Exclude<ExtractPathExpressions<T[P][number]>, keyof number | keyof string>}`
+      : T[P] extends { [x: string]: any }
+      ? `${P}${Sep}${ExtractPathExpressions<T[P]>}` | P
+      : P]: string;
+  },
+  symbol
+>;
+
```

Comment:
I think we don't need this type anymore

Reply:
I see that in this file, we use this type, but lets change the `fieldKey` type to just string.
Also remove the comment:
```
// eslint-disable-next-line @typescript-eslint/ban-ts-comment
// @ts-ignore
```